arrow-site/docs/docs/quickstart/README.md

```diff
@@ -7,28 +7,199 @@ permalink: /core/
 
 Λrrow is a library for Typed Functional Programming in Kotlin.
 
-Arrow aims to provide a [*lingua franca*](https://en.wikipedia.org/wiki/Lingua_franca) of interfaces and abstractions across Kotlin libraries.
-For this, it includes the most popular data types, type classes and abstractions such as `Option`, `Either`, `Functor`, `Applicative`, `Monad` to empower users to write pure FP apps and libraries built atop higher order abstractions.
+Arrow complements the Kotlin standard library and KotlinX Coroutines libraries with functional patterns that favor composition, effect control, and immutability.
+
+Arrow is composed of 4 main modular libraries.
 
-Use the list below to learn more about Λrrow's main features.
+## [Core]({{ '/' | relative_url }})
+Arrow Core includes types such as [`Either`]({{ '/apidocs/arrow-core-data/arrow.core/-either/' | relative_url }}), [`Validated`]({{ '/apidocs/arrow-core-data/arrow.core/-validated/' | relative_url }}) and many extensions to [`Iterable`]({{ '/apidocs/arrow-core-data/arrow.core/kotlin.collections.-iterable/' | relative_url }}) that can be used when implementing [error handling patterns]({{ '/patterns/error_handling/' | relative_url }}).
+Core also includes the base continuation effects system which includes patterns to remove callbacks and enable controlled effects in direct syntax. Some applications of the effect system reduce boilerplate and enable direct syntax include [monad comprehensions and computation expressions]({{ '/patterns/monad_comprehensions/' | relative_url }}).
 
-- [Patterns]({{ '/patterns/glossary/' | relative_url }}): tutorials and approaches to day-to-day challenges using FP
-- [Libraries]({{ '/quickstart/libraries/' | relative_url }}): all the libraries provided by Λrrow
-- [Effects]({{ '/effects/async/' | relative_url }}): interfacing with external systems
-- [Optics]({{ '/optics/iso/' | relative_url }}): inspecting and modifying data structures
+## [Fx]({{ '/fx/' | relative_url }})
+Arrow Fx is a full-featured, high-performance, asynchronous framework that brings functional operators to Kotlin's `suspend` functions.
+By leveraging the power of KotlinX Coroutines and the compiler support for CPS transformations, Arrow Fx results in optimal async programs with increased throughput and decreased allocations.
 
-## Curated external links
+## [Optics]({{ '/optics/' | relative_url }})
+Arrow Optics provides an automatic DSL that allows users to use `.` notation when accessing, composing and transforming deeply nested immutable data structures.
+Optics offers additionally all the base types such as [Lens]({{ '/optics/lens/' | relative_url }}), [Prism]({{ '/optics/prism/' | relative_url }}) and others from which we can generalize accessing and traversing deep values in sealed and data classes models.
 
-- [Projects and Examples]({{ '/quickstart/projects/' | relative_url }})
-- [Media](https://media.arrow-kt.io)
+## [Meta]({{ '/meta/' | relative_url }})
```

Comment:
```suggestion
## [Meta](https://meta.arrow-kt.io)
```

Meta is still using a different bucket and subdomain. It should be move as well :memo: